bears/python/PycodestyleBear.py

```diff
@@ -35,7 +35,7 @@ def create_arguments(
         :param max_line_length:
             Limit lines to this length.
         """
-        arguments = [r"--format='%(row)d %(col)d %(code)s %(text)s'"]
+        arguments = [r"--format='%(row)d %(col)d %(code)s %(text)s"]
```

Comment:
But the original is `r"--format='...'"` with two single quotes.
I suspect the problem is elsewhere...?  but maybe I dont understand correctly.
The issue says `Fix the regex of PycodestyleBear.`